packages/frontend/package.json

```diff
@@ -60,10 +56,8 @@
         "react": "^17.0.2",
         "react-ace": "^9.5.0",
         "react-beautiful-dnd": "^13.1.0",
-        "react-color": "^2.19.3",
         "react-copy-to-clipboard": "^5.0.4",
         "react-countdown": "^2.3.5",
-        "react-csv": "^2.2.2",
```

Comment:
we might add it again if we od the donwload csv from embeds in the frontend.

Reply:
sounds good!